src/site/content/en/metrics/user-centric-performance-metrics/index.md

```diff
@@ -0,0 +1,194 @@
+  to user interaction.
+- **Visual stability:** do elements on the page shift in ways that users don't
+  expect and potentially interfere with their interactions?
+- **Smoothness:** do transitions and animations render at a consistent frame
+  rate and flow fluidly from one state to the next?
+
+Given all the above types of performance metrics, it's hopefully clear that no
+single metric is sufficient to capture all the performance characteristics of a
+page.
+
+## Important metrics to measure
+
+- **[First contentful paint (FCP)](/fcp/):** measures the time from when the
+  page starts loading to when any part of the page's content is rendered on the
+  screen. _([lab](#in-the-lab), [field](#in-the-field))_
+- **[Largest contentful paint (LCP)](/lcp/):** measures the time from when the
+  page starts loading to when the largest text block or image element is
+  rendered on the screen. _([lab](#in-the-lab), [field](#in-the-field))_
+- **[First input delay (FID)](/fid/):** measures the time from when a user first
+  interacts with your site (i.e. when they click a link, tap a button, or use a
+  custom, JavaScript-powered control) to the time when the browser is actually
+  able to respond to that interaction. _([field](#in-the-field))_
+- **[Time to Interactive (TTI)](/tti/):** measures the time from when the page
+  starts loading to when it's visually rendered, its initial scripts (if any)
+  have loaded, and it's capable of reliably responding to user input quickly.
+  _([lab](#in-the-lab))_
+- **[Total blocking time (TBT)](/tbt/):** measures the total amount of time
+  between FCP and TTI where the main thread was blocked for long enough to
+  prevent input responsiveness. _([lab](#in-the-lab))_
+- **[Cumulative layout shift (CLS)](/cls/):** measures the cumulative score of
+  all layout shifts that occur between FCP and when the page's [lifecycle
+  state](https://developers.google.com/web/updates/2018/07/page-lifecycle-api)
+  changes to hidden. _([lab](#in-the-lab), [field](#in-the-field))_
+
+While this list includes metrics measuring many of the various aspects of
+performance relevant to users, it does not include everything (e.g. runtime
+responsiveness and smoothness are not currently covered).
+
+In some cases, new metrics will be introduced to cover missing areas, but in
+other cases the best metrics are ones specifically tailored to your site.
+
+## Custom metrics
+
+The performance metrics listed above are good for getting a general
+understanding of the performance characteristics of most sites on the web.
+They're also good for having a common set of metrics for sites to compare their
+performance against their competitors.
+
+However, there may be times when a specific site is unique in some way that
+requires additional metrics to capture the full performance picture. For
+example, the Largest Contentful Paint metric is intended to measure when a
```

Comment:
Here it is noted as 'Largest Contentful Paint' but above as 'Largest contentful paint' -> inconsistent casing.

Reply:
Thanks, updated.